parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -310,7 +310,7 @@ loop = do
         typeNotFound' = respond . TypeNotFound'
         termNotFound = respond . TermNotFound
         termNotFound' = respond . TermNotFound'
-        nameConflicted src tms tys = respond (NameAmbiguous hqLength src tms tys)
+        nameConflicted src tms tys = respond (DeleteNameAmbiguous hqLength src tms tys)
```

Comment:
Because it included this specialized output in OutputMessages:
https://github.com/unisonweb/unison/pull/1401/files#diff-85f642239bcbd3c585df34fdc7c38deeR824-R827
I thought the name `NameAmbiguous` was ambiguous, so I renamed to to deleteNameAmbiguous.  Now I see that the name `nameConflicted` is ambiguous, but whatever.